src/ucp/api/ucp.h

```diff
@@ -1310,6 +1311,11 @@ typedef struct ucp_worker_params {
     * using @ref ucp_conn_request_query.
     */
     uint64_t                client_id;
+
+    /**
+     * Align this offset of the active message data.
```

Comment:
refer to the field that specifies the alignment. also what is the default if not set?